README.md

```diff
@@ -0,0 +1,354 @@
+Introduction
+------------
+
+Provides a Web server for symbolicating Firefox stacks. It matches PC addresses
+to modules in memory and looks up the corresponding function names in
+server-side symbol files (.SYM files).
+
+This project is intended as a drop-in replacement for
+[Snappy Symbolication Server](https://github.com/mozilla/Snappy-Symbolication-Server/)
+
+Quick Start Guide
+-----------------
+
+1. Install [Python 2.7](https://www.python.org/downloads/) if it is not already
+   installed
+2. Install dependencies:
+   `python -m pip install tornado futures python-memcached psutil`
+3. Install [memcached](http://www.memcached.org/downloads)
+4. Copy/create a configuration file and set values appropriately. See
+   [Configuration File](#configuration-file)
+5. Run `python quickstart.py -c [configuration file]`
+
```

Comment:
The documentation that exists now documents options for how to configure quickstart not to start memcached. However, since this may be commonly desired functionality, I added a bit to the quickstart section of the documentation on which configuration options are relevant in this scenario.